addons/web_editor/static/lib/odoo-editor/test/spec/editor.test.js

```diff
@@ -2462,6 +2462,23 @@ X[]
                     contentAfter: '<p>a[]l</p>',
                 });
             });
+            it('should remove a table if completely selected', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: unformat(
```

Comment:
Thanks for the test :+1: Since your commit message says the issue also occured when the table was at the _end_, it would be good to make a test for that case too.